diagnostics-app/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "diagnostics-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~12.0.2",
+    "@angular/cdk": "^12.0.3",
+    "@angular/common": "~12.0.2",
+    "@angular/compiler": "~12.0.2",
+    "@angular/core": "~12.0.2",
+    "@angular/forms": "~12.0.2",
+    "@angular/material": "^12.0.3",
+    "@angular/platform-browser": "~12.0.2",
+    "@angular/platform-browser-dynamic": "~12.0.2",
+    "@angular/router": "~12.0.2",
+    "@angular/service-worker": "~12.0.2",
+    "rxjs": "~6.6.0",
```

Comment:
I don't see that it's used somewhere in the code. Do you need it?

Reply:
Same as above. Yes, all the dependencies that are listed automatically are used in the code. They're either imported in `app.module.ts` or will be needed very soon.

Reply:
We should add them when we need them.

Reply:
I had a doubt regarding `rxjs` not being used directly. It is being used as a dependency by `@angular/material`

```
"peerDependencies": {
    "@angular/animations": "^12.0.0 || ^13.0.0-0",
    "@angular/cdk": "12.0.3",
    "@angular/core": "^12.0.0 || ^13.0.0-0",
    "@angular/common": "^12.0.0 || ^13.0.0-0",
    "@angular/forms": "^12.0.0 || ^13.0.0-0",
    "rxjs": "^6.5.3"
  }
 ```

So all of them are essential in this PR. 👍🏻

Reply:
Thanks for double check!